### PR TITLE
chore: fresh translations

### PR DESCRIPTION
**Pull request by mroz22**
lot of changes in en.json so we probably should backport these into messages.ts later

### CODE REVIEW

File: packages/suite-data/files/translations/cs.json [hunk 1]
```diff
@@ -442,9 +442,9 @@
   "TR_COINMARKET_NO_OFFERS_AUTORELOADING_IN": "Automatické obnovení za ",
   "TR_COINMARKET_NO_OFFERS_BACK_BUTTON": "Zpět na obchod",
   "TR_COINMARKET_NO_OFFERS_HEADER": "Žádné nabídky",
-  "TR_COINMARKET_NO_OFFERS_LOADING_FAILED_MESSAGE": "Omlouváme se, ale naši poskytovatelé nemohou z důvodu problémů s připojením poskytovat žádné nabídky. Obnovte prosím tuto stránku.",
+  "TR_COINMARKET_NO_OFFERS_LOADING_FAILED_MESSAGE": "Omlouváme se, ale naši poskytovatelé nemohou není z důvodu problémů s připojením poskytnout žádné nabídky. Obnovte prosím tuto stránku.\n",
```
Comment (matejkriz) [inline packages/suite-data/files/translations/cs.json]: @alex-jerechinsky `není` -> `nyní`

File: packages/suite-data/files/translations/cs.json [hunk 2]
```diff
@@ -729,17 +729,17 @@
...
   "TR_EXCHANGE_SWAP_SLIPPAGE_MINIMUM": "Minimální přijatá částka",
   "TR_EXCHANGE_SWAP_SLIPPAGE_NOT_IN_RANGE": "Skluz musí být v rozmezí 0,01 % - 50 %.",
   "TR_EXCHANGE_SWAP_SLIPPAGE_NOT_NUMBER": "Zadejte prosím číslo.",
-  "TR_EXCHANGE_SWAP_SLIPPAGE_NOT_SET": "Zadejte požadovaný skluz.",
+  "TR_EXCHANGE_SWAP_SLIPPAGE_NOT_SET": "Zadejte požadovanou přijatelnou odchylku kurzu.",
   "TR_EXCHANGE_SWAP_SLIPPAGE_OFFERED": "Výše swapové nabídky",
-  "TR_EXCHANGE_SWAP_SLIPPAGE_SUMMARY": "Souhrn skluzu",
-  "TR_EXCHANGE_SWAP_SLIPPAGE_TOLERANCE": "Tolerance skluzu",
+  "TR_EXCHANGE_SWAP_SLIPPAGE_SUMMARY": "Souhrn odchylky skluzu",
```
Comment (matejkriz) [inline packages/suite-data/files/translations/cs.json]: @alex-jerechinsky `skluzu` -> `kurzu`

File: packages/suite-data/files/translations/cs.json (hunk 1 above)
Comment (jerechinsky) [reply packages/suite-data/files/translations/cs.json]: thank you, fixed

File: packages/suite-data/files/translations/cs.json (hunk 2 above)
Comment (jerechinsky) [reply packages/suite-data/files/translations/cs.json]: thank you, fixed

**Comment by mroz22** [comment]
added "fixup" commit. let's squash and merge after CI pass if your are ok with changes.

**Review by jerechinsky — APPROVED** [review]

**Pull request merged**